docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -516,6 +516,29 @@ The Iceberg JDBC catalog is supported for the Iceberg connector.  At a minimum,
 database besides PostgreSQL, a JDBC driver jar file must be placed in the plugin
 directory.
 
+:::{list-table} JDBC catalog configuration properties
+:widths: 40, 60
+:header-rows: 1
+
+* - Property name
+  - Description
+* - `iceberg.jdbc-catalog.driver-class`
+  - JDBC driver class name.
```

Comment:
Can we mention that a package prefix is required?

Reply:
How about adding a full example as well. Or even multiple.